app/services/edit_edition_service.rb

```diff
@@ -13,9 +13,26 @@ def call
     edition.assign_attributes(
       attributes.merge(last_edited_by: user, last_edited_at: Time.current),
     )
+
+    determine_political
+    determine_government
```

Comment:
So all content is going to be associated with a government, regardless of whether it's political or not?

Reply:
Yes - that's how Whitehall has done it.